packages/tofu/src/findGlobals.js

```diff
@@ -76,13 +82,26 @@ function findGlobals(ast) {
 
   return globals
 
-  function saveGlobal(path, name = path.node.name) {
-    // init entry if needed
-    if (!globals.has(name)) {
-      globals.set(name, [])
+  /**
+   *
+   * @param {import('@babel/traverse').NodePath<import('@babel/types').Identifier|import('@babel/types').ThisExpression>} path
+   * @param {string} [name]
+   */
+  function saveGlobal(path, name) {
+    if ('name' in path.node) {
+      name = path.node.name
+    }
+    if (name) {
+      // init entry if needed
+      if (!globals.has(name)) {
+        globals.set(name, [])
+      }
+      // append ref
+      const refsForName =
+        /** @type {import('@babel/traverse').NodePath<import('@babel/types').Identifier|import('@babel/types').ThisExpression>[]} */ (
```

Comment:
This type is being reused here a bunch, I'd be happy if we could name it something. The annoyance caused by repeating it is growing beyond the potential annoyance of it being exported.